src/ipyvizzu/chart.py

```diff
@@ -20,8 +20,8 @@ class Chart:
 
     # pylint: disable=too-many-instance-attributes
 
-    VIZZU: str = "https://cdn.jsdelivr.net/npm/vizzu@0.9/dist/vizzu.min.js"
-    """A variable for storing the default url of vizzu package."""
+    VIZZU: str = VIZZU
```

Comment:
I know that Python handles this right, but this line is weird. Can you rename imported `VIZZU` to anything else like `VIZZU as _VIZZU` to clarify the `VIZZU` name in this file?